src/Tribe/Attendee_Registration/View.php

```diff
@@ -148,19 +152,14 @@ public function display_attendee_registration_shortcode() {
 		wp_localize_script(
 			'event-tickets-registration-page-scripts',
 			'TribeCurrency',
-			[
-				'formatting' => json_encode( $currency )
-			]
+			[ 'formatting' => json_encode( $currency ) ]
```

Comment:
:warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: 59583ea12c7488d1fb37b4e3acf155ac</sub>